src/fluree/db/util/context.cljc

```diff
@@ -198,14 +198,25 @@
 (defn txn-context
   "Remove the fluree context from the supplied context."
   [txn]
-  (validate-txn-context txn)
-  (let [supplied-context (->> (get txn "@context" (:context txn))
-                              (util/sequential)
-                              (remove #{"https://ns.flur.ee"}))]
+  (let [supplied-context (when (or (contains? txn :context)
+                                   (contains? txn "@context"))
+                              (->> (get txn "@context" (:context txn))
+                                   (util/sequential)
+                                   (remove #{"https://ns.flur.ee"})))]
```

Comment:
what happens if the user also supplies the fluree context?

Reply:
We remove it. This is the status quo, though, and this commit doesn't change it. Once we properly remove default context this hack will no longer be necessary.